### PR TITLE
Fix outdated README output

### PR DESCRIPTION
**Pull request by ostrolucky**

### CODE REVIEW

**Comment by bpolaszek** [comment]
Didn't notice this! :sweat_smile:

File: README.md [hunk 1]
```diff
@@ -77,24 +77,24 @@ Array
     [2] => Array
         (
             [hair] => blond
-            [eyes] => brown
+            [eyes] => grey
```
Comment (bpolaszek) [inline README.md]: Actually it's blue.

Comment (ostrolucky) [reply README.md]: blue is already in index 0

Comment (bpolaszek) [reply README.md]: My bad, my coffee wasn't hard enough this morning! :sweat_smile:

**Pull request merged**